public class Selection_sort {
    public static  void selection_sort(int[] a){
        for (int i = 0; i < a.length; i++){
            int small = i;
            for (int j = i+1; j < a.length; j++){
                if (a[small] > a[j]){
                    small = j;
                }
            }
            int temp = a[i];
            a[i] = a[small];
            a[small] = temp;    
        }
    }

    public static void printArr(int[] a){
        for (int elem : a){
            System.out.print(elem + " ");
        }
    }

    public static void main(String[] args){
        int[] a = {65,26,13,23,12};
        System.out.println("Arreglo antes de ser ordenado: ");
        printArr(a);
        selection_sort(a);
        System.out.println("\nArreglo despues de ser ordenado: ");
        selection_sort(a);
        printArr(a);
    }
    
}
